tests/functional_test_base.go

```diff
@@ -437,3 +437,7 @@ func (s *FunctionalTestBase) registerArchivalNamespace(archivalNamespace string)
 	)
 	return err
 }
+
+func (s *FunctionalTestBase) overrideDynamicConfig(setting dynamicconfig.GenericSetting, value any) {
```

Comment:
I'm wondering if this should be public so it can be used from other packages, e.g. the ndc/xdc tests. Currently some xdc tests do override dynamic config, but they manage multiple clusters so they call temporalImpl.OverrideDCValue directly, which is fine.

Reply:
No tests in other packages were embedding FunctionalTestBase so far. But you are right, it should be possible.